main.js

```diff
@@ -205,60 +223,69 @@ function clearFormFields() {
   studyButton.checked = false;
   meditateButton.checked = false;
   exerciseButton.checked = false;
+  showImg();
+  hideActiveImg();
+};
+
+function showImg() {
   showElement(meditateImg);
   showElement(studyImg);
   showElement(exerciseImg);
+};
+
+function hideActiveImg() {
   hideElement(meditateImgActive);
   hideElement(studyImgActive);
   hideElement(exerciseImgActive);
-}
+};
 
 function countdown() {
   currentActivity.startTimer();
   document.getElementById("startButton").disabled = true;
-}
+};
 
-function showComplete(){
+function showComplete() {
   startTimerButton.innerText = 'COMPLETE!';
   showElement(logActivityButton);
 }
 
+//localStorage functions
```

Comment:
These labels are a nice addition for this project since there's a lot going on. Way to show empathy for your future selves and other devs.